### ISSUE TITLE
No Intellisense?

### ISSUE THREAD

**Issue by walles** [microsoft/vscode-makefile-tools#332]
I'm on Makefile Tools `v0.5.0`, VS Code `1.67.2`, macOS Big Sur 11.6.6, AMD64.

After working around #327 I can't notice the extension doing anything at all.

Does it require the C/C++ extension to be installed? I don't have that.

The docs indicates the other extension is a requirement:

![Skärmavbild 2022-06-01 kl  13 55 34](https://user-images.githubusercontent.com/158201/171399095-c71f5b01-b3bc-482c-8229-6bd502dcef12.png)

**Comment by bobbrow** [comment]
> Does it require the C/C++ extension to be installed? I don't have that.

Yes, to get IntelliSense in your source files, you need to install the Microsoft C/C++ extension.  You do not need it to build your project, but you will also want the C/C++ extension if you plan to debug your project in VS Code since it also provides the debugger.

**Comment by walles** [comment]
> you will also want the C/C++ extension if you plan to debug your project in VS Code since it also provides the debugger.

We're using Makefiles in a Go project. The Go VSCode addon provides debugging support.

> to get IntelliSense in your source files, you need to install the Microsoft C/C++ extension.

But I just want Makefiles, not C/C++. I don't see the connection?

**Comment by bobbrow** [comment]
Are you asking for IntelliSense in the Makefile, not in the source code then?  We don't have any immediate plans to add a full language server for Makefile, but may do some partial support such as for #208.

**Comment by bobbrow** [comment]
To clarify, there is currently no IntelliSense for the actual Makefiles themselves.  Our extension provides the build configurations to the C/C++ extension so that you can get IntelliSense for C/C++ files.  Since you are using Go, it sounds like that is less interesting for you.

**Comment by walles** [comment]
I just want `Makefile` support, nothing else. Based on the name ("Makefile Tools") I assumed this extension would improve my `Makefile` editing experience.

But given what you just said I think I'm simply not in the target audience for the Makefile Tools.

**Comment by bobbrow** [comment]
Yes, sorry, we developed this extension with C++ in mind.  We do IntelliSense configuration, build, and debug integration for C/C++ projects.  We may expand our charter to include a language server for Makefiles in the future, but unfortunately we're not at a place where we can fund it yet.

**Comment by Curiouspaul1** [comment]
😭I need thiss
